.github/workflows/main.yml

```diff
@@ -76,7 +76,7 @@ jobs:
       - name: Run tests
         env:
           DATABASE_URL: postgres://user:password@localhost/jobserver
-          GITHUB_TOKEN: empty
```

Comment:
The GITHUB_TOKEN used in CI is different to the one we're using locally and in prod, so these two yaml files shouldn't be changed.

Reply:
Ah thanks for letting me know, I was a bit unsure of this change